embrace_mobile/README.md

```diff
@@ -0,0 +1,91 @@
+each and every user experience, Embrace extracts powerful insights to fuel your growth.
+
+Post integration, Embrace provides dashboards that track key mobile health metrics. For any regression, you can inspect 
+the full details of every affected user session without having to manually reproduce it. 
+
+### Embrace Datadog Dashboard
+
+The Datadog dashboard connects client-side Embrace data for crashes and networking to your server-side data. 
+
+![Embrace Datadog Dashboard][2]
+
+#### Crashes
+
+Monitor crash trends and inspect stack traces for every crash directly within Datadog. For more context, review full
+user session details within Embrace.
+
+![Crashes Side Panel][3]
+
+#### Networking
+Embrace collects every network call, even the ones that never make it to the servers. You will see all errors on your
+endpoints, including 4xxs, 5xxs, timeouts, and connection errors.
+
+![Network Side Panel][4]
+
+---
+
+From Datadog, immediately navigate to the actionable data and insights needed to optimize mobile experiences within the
+Embrace platform. These include:
+
+#### User Session Replays
+Experience any user session without the hassle of reproducing. Replay all the technical and behavioral details in one
+traversable visual so that any team member can immediately answer any question or complaint.
+
+![Embrace Session Timeline][5]
+
+#### App Performance
+Understand every cause of broken experiences to correctly measure and improve mobile app health. Beyond crashes,
+optimize performance to remove slow or frozen areas that lead to force quits and abandonment of key user flows. 
+
+![Embrace App Performance][6]
+
+#### Dashboards
+Track adoption, performance, stability, engagement, and monetization with real-time dashboards for the metrics that 
+matter to your business. Create individual or team dashboards to hone in on the features or experiments they care about.
+
+![Embrace Dashboard][7]
+
+#### Error Logging
+Set session and log properties to allow powerful filtering to isolate the root cause. Easily uncover patterns across 
+device, version, OS, region, user segment, and event attributes so that your team knows the business impact and 
+remediation priority of issues. 
+
+#### Proactive Alerting
+Be the first to know about regressions in the performance and stability of your features and releases. Whether the 
+failure is in your code, your backend, or a bad actor third-party SDK or vendor, Embrace notifies you in real-time
+so you can take action before your users complain.
+ 
+## Setup
+
+1. Start a free trial with Embrace and follow our [documentation here][8]. **You’ll need to 
+   complete a basic integration before seeing metrics within Datadog.**
+1. After completion of the Embrace integration, you can come back to Datadog to connect both platforms.
+1. Create a “New Dashboard” in Datadog and select the Embrace widget.
+1. Authenticate and connect your Embrace account to Datadog by simply logging in with your credentials.
```

Comment:
This step seems like it should go above the create a new dashboard and select embrace widget step.